main/lsp/LSPIndexer.cc

```diff
@@ -310,6 +319,12 @@ LSPFileUpdates LSPIndexer::commitEdit(SorbetWorkspaceEditParams &edit, std::vect
     if (!update.canTakeFastPath) {
         update.updatedGS = initialGS->deepCopy();
         pendingTypecheckUpdates = update.copy();
+        if (pendingTypecheckLatencyTimer != nullptr) {
```

Comment:
The following two blocks of changes "save" clones from the latency timers from slow paths so that, if they get canceled, the canceler can use it as its latency timer.